packaging/rpm/flightctl.spec

```diff
@@ -152,6 +153,7 @@ fi
 
 %files agent -f licenses.list
     %license LICENSE
+    %dir /etc/flightctl
```

Comment:
Today the config-dir is variable and created by the agent; do we need both?

Reply:
It's default value is
```go
DefaultConfigDir = "/etc/flightctl"
```
and it is mentioned in getting-started.md, managing-devices.md, provisioning-devices.md
So it should be created by default.
WDYT?